README.md

```diff
@@ -1,2 +1,14 @@
 # couchdb-cluster-admin
 utility for managing multi-node couchdb 2.x clusters
+
+
+# Help estimating shard allocation
```

Comment:
When you were looking this up did you see any recommendations on number of shards to use? We may see some improvements by using fewer shards for some indicies or more for synclogs

Reply:
I didn't look at that at all, no. We're currently on the default of 8 for all the dbs. I think for my current purposes this is totally fine; that kind of tuning may actually make a difference on the likes of ICDS, though.

Reply:
Yeah. I was specifically thinking of ICDS. though http://docs.couchdb.org/en/2.1.0/cluster/sharding.html#reshard-no-preshard is the only mention I've seen of it (and it doesn't really give you a way to figure out how many to use)